core/metacling/test/TClingLoadUnloadFileTests.cxx

```diff
@@ -45,11 +51,11 @@ void compile_n_macros(const std::string &basename, unsigned int n)
 
 void load_unload(const std::string &basename, unsigned int n)
 {
-   std::string library_so{basename + "_" + std::to_string(n) + "_cpp.so"};
+   std::string library_so{basename + "_" + std::to_string(n) + "_cpp" + suffix};
 
    for (int i = 0; i < 100; i++) {
-      gInterpreter->LoadFile(library_so.c_str());
-      gInterpreter->UnloadFile(library_so.c_str());
+      EXPECT_EQ(gInterpreter->LoadFile(library_so.c_str()), 0) << "Failed to load " << library_so << std::endl;
```

Comment:
Can you do this outside a `TEST`? I.e.: did you try to make sure that this fails correctly?

Reply:
Obviously I did!